src/engine/cachingreader/cachingreaderworker.cpp

```diff
@@ -130,7 +123,6 @@ void CachingReaderWorker::loadTrack(const TrackPointer& pTrack) {
     }
 
     // Unload the track
-    m_readableFrameIndexRange = mixxx::IndexRange();
```

Comment:
Why can we still remove this? Maybe we cannot anymore because of a new assertion the cachingreader.c (not verified)